src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -23,4 +23,125 @@
           short-summary: 'Specify what should happen if an entity already exists for the specified PartitionKey and RowKey.'
         - name: --timeout
           short-summary: The server timeout, expressed in seconds.
-"""
\ No newline at end of file
+"""
+
+helps['storage'] = """
+    type: group
+    short-summary: Durable, highly available, and massively scalable cloud storage
+"""
+helps['storage account'] = """
+    type: group
+    short-summary: Commands to manage your Storage accounts
+"""
+helps['storage account keys'] = """
+    type: group
+    short-summary: Commands to manage your Storage aaccount keys
+"""
+helps['storage blob'] = """
+    type: group
+    short-summary: REST-bsed object storage for unstructured data
+"""
+helps['storage blob copy'] = """
+    type: group
+    short-summary: Commands to manage your blob copy operations
+"""
+helps['storage blob lease'] = """
+    type: group
+    short-summary: Commands to manage leases of your storage blob
+"""
+helps['storage blob metadata'] = """
+    type: group
+    short-summary: Commands to manage your blob metadata
+"""
+helps['storage blob service-properties'] = """
+    type: group
+    short-summary: Commands to view Storage blob service properties
```

Comment:
>Commands to view Storage blob service properties [](start = 19, length = 48)

Hmm... a subgroup with a single command.. this should probably be a hyphenated command "show-service-properties" instead of a group.